README.md

```diff
@@ -3,26 +3,29 @@
 
 # Neomake
 
-Neomake is a plugin for asynchronous `:make` using
-[Neovim's](http://neovim.org/) job-control functionality. It is inspired by
-the excellent Vim plugins [Syntastic](https://github.com/scrooloose/syntastic)
-and [dispatch.vim](https://github.com/tpope/vim-dispatch).
+Neomake is a plugin that asynchronously runs programs using
+[Neovim]'s job-control functionality. It is intended to
+replace Vim's built-in `:make` command and provides functionality similar to
+plugins like [syntastic] and [dispatch.vim]. It is primarily used to run code
+linters and compilers from within Vim, but can be used to run any program.
 
 **This plugin also works in ordinary Vim, but without the asynchronous
 benefits.**
 
-The minimum Neovim version supported by Neomake is `NVIM
-0.0.0-alpha+201503292107` (commit `960b9108c`).
-The minimum Vim version supported by Neomake is 7.4.503 (although if you don't
-use `g:neomake_logfile` older versions will probably work fine as well).
+## Requirements
+
+The minimum [Neovim] version supported by Neomake is
+`NVIM 0.0.0-alpha+201503292107` ([commit 960b9108c]). The minimum Vim version
+supported by Neomake is 7.4.503 (although if you don't use `g:neomake_logfile`
+older versions will probably work fine as well).
 
 ## How to use (basic)
 
 Just set your `makeprg` and `errorformat` as normal, and run:
 
     :Neomake!
 
-If your makeprg can take a filename as an input, then you can run `:Neomake`
+If your `makeprg` can take a filename as an input, then you can run `:Neomake`
```

Comment:
The documentation for `:h :Neomake` says
```
:Neomake [makers]       Run a make command with the current file as input. If
                        no makers are specified, the default makers for the
                        current |filetype| are used. See |neomake-configuration|
                        for more on makers.
```

The first argument you pass the _command_ is the maker to run.
The maker's `maker.exe` is passed the filename as an arg along with the rest of the args in `maker.args`
So the docs aren't wrong, but this sentence fails to serve its purpose (distinguishing the purpose of `:Neomake` from `:Neomake!`)

Perhaps I'll rewrite this one entirely -- "If your `makeprg` should run on the file in the current buffer, then you can use `:Neomake`" (as opposed to running `:Neomake!` to execute the global maker in the CWD)